### DISCUSSION TITLE
Passaggio al nuovo anno 2024/2025 e recupero debiti

### DISCUSSION THREAD

**Discussion by luigirotondi2009** [iisgiua/giuaschool#735] [Richieste di aiuto]
Salve,  la prossima settimana dovrebbe uscire l'aggiornamento per il recupero dei debiti e passaggio al nuovo anno, giusto?
È consigliabile prima svolgere lo scrutinio del giudizio sospeso e poi effettuare il passaggio d'anno o viceversa?

**Comment by trinko** [comment]
Non so ancora una data precisa per il prossimo l'aggiornamento.
La procedura da seguire comunque è:
- completare tutti gli scrutini con il recupero dei debiti;
- poi, procedura di archiviazione dei dati;
- poi procedura di passaggio al nuovo anno.

**Reply by luigirotondi2009** [reply]
Come possiamo procedere per svolgere il recupero del debito, selezionando il voto sul registro?

**Reply by trinko** [reply]
> Come possiamo procedere per svolgere il recupero del debito, selezionando il voto sul registro?

Aspetta la nuova versione.

**Reply by luigirotondi2009** [reply]
La nuova versione 1.5.9, quando dovrebbe uscire?

**Reply by trinko** [reply]
> La nuova versione 1.5.9, quando dovrebbe uscire?

Penso di farcela per la fine di questa settimana

**Reply by luigirotondi2009** [reply]
Cosa comprende la nuova versione?

**Reply by trinko** [reply]
> Cosa comprende la nuova versione?

La procedura dello scrutinio degli alunni con sospensione del giudizio: correzioni e inserimento proposta di voto.

**Reply by luigirotondi2009** [reply]
C'è una guida già disponibile o dobbiamo aspettare il nuovo aggiornamento questa settimana?
